src/main/java/de/chojo/repbot/listener/StateListener.java

```diff
@@ -93,7 +93,7 @@ public void onChannelDelete(@NotNull ChannelDeleteEvent event) {
     @Override
     public void onEmojiRemoved(@NotNull EmojiRemovedEvent event) {
         var guildSettings = guilds.guild(event.getGuild()).settings();
-        guildSettings.thanking().reactions().remove(event.getEmoji().getId());
+//        guildSettings.thanking().reactions().remove(event.getEmoji().getId());
```

Comment:
Why is this commented?

Reply:
Good catch, that should not have been included in here and I overlooked that while making the commit